codecov_cli/parsers/base.py

```diff
@@ -0,0 +1,60 @@
+from datetime import datetime, timedelta
+from typing import List
+
+
+class ParsingError(Exception):
+    ...
+
+
+class Test(object):
+    def __init__(self, name: str, status: bool, duration: timedelta):
+        self.name = name
+        self.status = status
```

Comment:
I think `status` should be an Enum rather than a bool.
Or change it to "passed" or "successful"